UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Data/GasMix.cs

```diff
@@ -511,7 +545,7 @@ public override string ToString()
 
 		public void Clear()
 		{
-			Temperature = AtmosDefines.SPACE_TEMPERATURE;
+			Temperature = 0;
```

Comment:
why zero? I think the constant was a negative, isn't it?

Reply:
constant isn't a negative, also you can't have negative temperature

Reply:
I'm just asking. Want to understand the possible consequences of changing to zero.

Reply:
I had a quick look through code and there shouldn't be any divide by zero, the main reason I did it was because this means you could technically farm a little bit of energy from it, but it shouldn't change it either way

Reply:
basically if the code allows zero , shouldn't be afraid to use it